src/storage/mod.rs

```diff
@@ -1471,9 +1471,9 @@ mod tests {
         }
     }
 
-    fn expect_ok_callback<T>(done: Sender<i32>, id: i32) -> Callback<T> {
+    fn expect_ok_callback<T: Debug>(done: Sender<i32>, id: i32) -> Callback<T> {
         Box::new(move |x: Result<T>| {
-            assert!(x.is_ok());
+            x.unwrap();
```

Comment:
Change to `unwrap()` so that when there are `Result::Err` we can learn what error it is from stdout.

Reply:
why not using expect?

Reply:
@siddontang Seems that the message `unwrap()` gives is verbose enough, there is no need to write a more verbose version like `x.expect("");`.